manim/mobject/svg/text_mobject.py

```diff
@@ -1387,3 +1387,75 @@ def extract_color_tags(self):
 
     def __repr__(self):
         return f"MarkupText({repr(self.original_text)})"
+
+
+@contextmanager
+def register_font(font_file: typing.Union[str, Path]):
+    """Context Manager which adds a font
+    file to Pango's search path.
+    This searches for the font_file at various places.
+
+    The order it searches it described below.
+
+    1. Absolute path.
+    2. In ``assets/fonts`` folder.
+    3. In ``font/`` folder.
+    4. In the same directory.
+
+    .. note ::
+
+        This method of adding font files also works with
+        :class:`CairoText`
+
+    .. important ::
+
+        This method isn't available for macOS. Using this
+        method on macOS will raise an :class:`AttributeError`.
```

Comment:
For consistency: please change `.. note ::` into a `Notes\n-----` section.

Please add the AttributeError to the `Raises` section.